docs/adding-a-new-xpi.md

```diff
@@ -26,6 +26,23 @@ git merge --allow-unrelated-histories template/master
 # fix conflicts, commit result
 ```
 
+### External repos
+
+What if I don't want to move my repository into mozilla-extensions?
+
+First, we need to make sure we meet the following requirements:
+
+- The repo needs to follow the security guidelines: branch protections, reviews, limit who is an admin. Private repos will need to invite the `mozilla-extensions/private-repo` team to enable automation.
```

Comment:
Unfortunately, teams can not cross organization boundaries. The external org will need to invite each of the bot accounts to the external org (at the least). Then, a human has to log into the bot accounts and accept the invitation within 7 calendar days. (I can outline a series of actions the external org will need to perform the setup in one login session by an org owner.)

It is a good idea to have both in a team for ease of administration. We did that for the pyup service, but the hassle of such administrative overhead was one of the drivers to discontinue the service.